Userland/Libraries/LibELF/Arch/aarch64/entry.S

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2021, Nico Weber <thakis@chromium.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+    .globl _invoke_entry
+    .hidden _invoke_entry
+    .type _invoke_entry,@function
```

Comment:
I don't know much about asm, any reason why these are indented like this?

Reply:
It's probably because that's how `gcc -S -o - foo.cc` indents it: Labels in column 0, everything else indented. I agree it's ugly though, and most .S files don't look like this. So I fixed this and added a commit that reindents the few files that didn't match this style.

Reply:
We're also pretty undecided on what comment style to use in .S files. I didn't touch that for now.